app/src/main/java/io/gnosis/safe/ui/settings/owner/OwnerSeedPhraseViewModel.kt

```diff
@@ -12,7 +12,42 @@ class OwnerSeedPhraseViewModel
     appDispatchers: AppDispatchers
 ) : PublishViewModel<ImportOwnerKeyState>(appDispatchers) {
 
-    fun validate(seedPhrase: String) {
+    fun validate(seedPhraseOrKey: String) {
+        if (isPrivateKey(seedPhraseOrKey)) {
+            validatePrivateKey(seedPhraseOrKey)
+        } else {
+            validateSeedPhrase(seedPhraseOrKey)
+        }
+    }
+
+    internal fun isPrivateKey(seedPhraseOrKey: String): Boolean {
```

Comment:
we could mark methods with relaxed visibility due to testing with 
@VisibleForTesting(otherwise = VisibleForTesting.PRIVATE)

Reply:
It doesn't seem to break the build. But it highlights non private usage:
![image](https://user-images.githubusercontent.com/246473/106141672-bbcdf300-6170-11eb-8e70-78fc2e959504.png)

Reply:
Done in: 5225b2e8bc8ab0730e628a210c4435c101221dbd